public/js/views/management.jsx

```diff
@@ -70,14 +54,9 @@ export default class Management extends Component {
     this.props.getUserSubscriptions();
   }
 
-  userSignIn = e => {
-    e.preventDefault();
-    this.props.userSignIn();
-  }
-
-  userSignOut = e => {
+  showSignOut = e => {
     e.preventDefault();
-    this.props.userSignOut();
+    this.props.showSignOut();
```

Comment:
I was unclear (highlighted the wrong line) the bit I'm talking about renaming is the hander func not the action prop.